google/resource_compute_vpn_tunnel.go

```diff
@@ -399,6 +406,9 @@ func resourceComputeVpnTunnelRead(d *schema.ResourceData, meta interface{}) erro
 	if err := d.Set("peer_ip", flattenComputeVpnTunnelPeerIp(res["peerIp"])); err != nil {
 		return fmt.Errorf("Error reading VpnTunnel: %s", err)
 	}
+	if err := d.Set("shared_secret", flattenComputeVpnTunnelSharedSecret(res["sharedSecret"])); err != nil {
```

Comment:
I feel like we weren't setting this before on purpose, weren't we?

Reply:
Totally - the merge removed the bit that ignored things with ignore_read set.  :(  Fixed and rerunning.